assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -150,7 +150,10 @@ export const GridItemButton = styled.button`
   ${themeHelpers.focusableOutlineCSS};
 `;
 
-export const Logo = styled.img`
+export const Logo = styled.img.attrs({
+  height: '96px',
+  width: '96px',
```

Comment:
```suggestion
  height: 96,
  width: 96,
```


These attributes are unit-less :-)